src/app/app.component.ts

```diff
@@ -63,4 +81,11 @@ export class AppComponent implements OnInit {
     });
   }
 
+  getExpConfig(): Observable<any> {
+    let domain = window.location.hostname;
+    domain = (domain.indexOf('127.0.0.1') !== -1 || domain.indexOf('localhost') !== -1) ? 'appdev.practera.com' : domain;
```

Comment:
We don't need to force using `appdev.practera.com`, we can just use `localhost` or `127.0.0.1`. It's actually better for testing